locations/spiders/davidstea.py

```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 from scrapy.spiders import SitemapSpider
 
-from locations.hours import OpeningHours
```

Comment:
This was unused so removed it.
This spider is working well after removing above import.